plop/dotnet/service.mjs

```
@@ -111,45 +141,38 @@ function dotnetServiceFactory (plop) {
-          RESOURCE_NAME: answers.RESOURCE_NAME,
-          RESOURCE_NAME_P: answers.RESOURCE_NAME_P,
-          RESOURCE_SCHEMA_VERSION: answers.RESOURCE_SCHEMA_VERSION,
-          TELEMETRY_SERVICE_NAME: answers.TELEMETRY_SERVICE_NAME,
-          DB_SERVER: answers.DB_SERVER,
-          DB_PORT: answers.DB_PORT,
-          DB_USER: answers.DB_USER,
-          DB_PASSWORD: answers.DB_PASSWORD,
-          DB_DATABASE: answers.DB_DATABASE
-        }
+        configData = questions.reduce((total, current) => {
+          total = { ...total, [current.name]: answers[current.name] }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)